docs/ug/src/main/jbake/content/attributes.conf

```diff
@@ -20,14 +20,14 @@
 // for the technology.  Used in config.inc.
 :TechnologyHomeEnv: JARPATH
 // Java SE version required.
-:SEversion: 9
+:SEversion: 8
 :JavaTestVersion: 5.0
-:jteFileName: <TS_HOME>/bin/ts.jte
-:jtxFileName: <TS_HOME>/bin/ts.jtx
-:TCKPackageName: jaftck-1.2.zip
+:jteFileName: <TS_HOME>/lib/ts.jte
+:jtxFileName: <TS_HOME>/lib/ts.jtx
+:TCKPackageName: jaftck-2.0.zip
```

Comment:
Should be "jakarta-activation-tck-2.0.0.zip"  as in https://download.eclipse.org/ee4j/jaf/tck/eftl/jakarta-activation-tck-2.0.0.zip

Reply:
fixed